external-crates/move/crates/move-compiler/src/shared/mod.rs

```diff
@@ -687,6 +691,10 @@ pub struct Flags {
     /// included only in tests, without creating the unit test code regular tests do.
     #[clap(skip)]
     keep_testing_functions: bool,
+
+    /// If set, all warnings are silenced
+    #[clap(skip = false)]
```

Comment:
What does skip mean here? Confused because `keep_testing_functions` is marked as skip

Reply:
This omits it from the CLI interface, etc.